README.md

```diff
@@ -1,87 +1,105 @@
-# Transparent Release of Binaries
+# The Transparent Release Project
+
+Transparent Release (not really the best name for the project) builds on the idea of binary
+transparency and expands in a few directions, as described below. 
+
+## Release Transparency
+
+The goal of release transparency is to generate and transparently publish ideally-verifiable metadata about a released binary. We refer to this kind of metadata as the endorsement statement. It can be used in scenarios where the identity of a binary must be verified before use. One prominent use case is remote attestation. See Oak’s remote attestation protocol for instance.
```

Comment:
ideally-verifiable -> ideally verifiable

Does it make sense to link [Oak's remote attestation protocol](https://github.com/project-oak/oak/tree/main/remote_attestation)